build.gradle

```diff
@@ -1,66 +1,112 @@
 plugins {
-    id "org.jetbrains.kotlin.jvm" version "1.3.30"
+    id "org.jetbrains.kotlin.jvm" version "1.3.31"
+    id "maven-publish"
 }
 
-apply plugin: 'kotlin'
-apply plugin: 'java'
-apply plugin: 'maven'
-apply plugin: 'idea'
-
 group = 'org.openbroker'
 version = '1.0-SNAPSHOT'
 description = 'Library for cloud events'
 
-defaultTasks 'test'
+wrapper.gradleVersion = '5.4.1'
+
+ext {
+    kotlinLanguageVersion = '1.3'
+    jdkVersion = JavaVersion.VERSION_1_8
+
+    defaultEncoding = 'UTF-8'
 
-project.ext {
-    kotlin_version = '1.3.30'
-    kotlin_api_version = '1.3'
-    jvm_version = '1.8'
+    ver = [
+        'kotlin' : '1.3.31',
+        'junit'  : '5.4.2',
+        'jackson': '2.9.9',
+    ]
+
+    servicePackage = 'org.openbroker.cloudevents'
 }
 
-apply from: "jar.gradle"
+sourceCompatibility = jdkVersion
+targetCompatibility = jdkVersion
 
 repositories {
+    maven {
+        url "https://jitpack.io"
+        content {
+            includeGroupByRegex 'com\\.(github|gitlab|gitee).*'
+        }
+    }
     jcenter()
     mavenCentral()
-    maven { url "https://dl.bintray.com/kotlin/kotlinx" }
 }
 
 dependencies {
-    // Kotlin
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
 
-    // Jackson
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version:'2.9.8'
-    implementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-kotlin', version:'2.9.8'
-    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version:'2.9.8'
+    implementation(
+        platform("com.fasterxml.jackson:jackson-bom:$ver.jackson"),
+        platform("org.jetbrains.kotlin:kotlin-bom:$ver.kotlin"),
+        "org.jetbrains.kotlin:kotlin-stdlib-jdk8",
+        "com.fasterxml.jackson.core:jackson-core",
+        "com.fasterxml.jackson.module:jackson-module-kotlin",
+        "com.fasterxml.jackson.datatype:jackson-datatype-jsr310",
+    )
 
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.4.0")
-    testRuntime("org.junit.jupiter:junit-jupiter-engine:5.4.0")
+    testImplementation(
+        platform("org.junit:junit-bom:$ver.junit"),
+        "org.junit.jupiter:junit-jupiter-api"
+    )
+    testRuntime(
+        "org.junit.jupiter:junit-jupiter-engine"
+    )
 }
 
 tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
     kotlinOptions {
-        languageVersion = kotlin_api_version
-        apiVersion = kotlin_api_version
-        jvmTarget = jvm_version
-        javaParameters = true   // Useful for reflection.
+        javaParameters = true
```

Comment:
Comment is not there, think its still unclear for people why you need to have this parameter.